src/test/java/duke/UiTest.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UiTest {
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+    private final Ui ui = new Ui();
+
+    @BeforeEach
+    public void setUp() {
+        System.setOut(new PrintStream(outputStreamCaptor));
+    }
+
```

Comment:
Should the test be just calling the function? Maybe you can call functions that will invoke the ui showing those messages, then see if your ui actuall respond.